moped-database/views/component_arcgis_online_view.sql

```diff
@@ -169,7 +169,11 @@ SELECT
     component_tags.component_tags,
     mpc.description AS component_description,
     mpc.interim_project_component_id,
-    coalesce(mpc.completion_date, plv.substantial_completion_date) AS substantial_completion_date,
+    CASE
+        WHEN mpc.phase_id IS null THEN plv.substantial_completion_date
+        WHEN mpc.phase_id IS NOT null AND mpc.completion_date IS null THEN null::timestamp with time zone
```

Comment:
TIL that `timestampz` and `timestamp with time zone` are the same thing by different names. Not sure which is an alias for the other, but it doesn't matter -- I think this is the most clear of the two. 🌹